src/sql/platform/azureBlob/common/azureBlobService.ts

```diff
@@ -0,0 +1,18 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { createDecorator } from 'vs/platform/instantiation/common/instantiation';
+import type * as mssql from 'mssql';
+
+export const SERVICE_ID = 'azureBlobService';
+
+export const IAzureBlobService = createDecorator<IAzureBlobService>(SERVICE_ID);
+
+export interface IAzureBlobService {
+	_serviceBrand: undefined;
+	/**
+	* Create shared access signature for blob container
```

Comment:
Please add doc comments for the parameters as well (use the ones I added to mssql.d.ts, feel free to update if you have better descriptions)